content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -31,52 +31,24 @@ but not production. Like all alpha features, this guide is subject to change.
 1. Virtual machines must have IP connectivity to the ingress gateway in the connecting mesh, and optionally every pod in the mesh via L3 networking if enhanced performance is desired.
 
 ## Prepare the guide environment
-
-1. Set the environment variables `"${ISTIO_DIR}"`, `"${WORK_DIR}"`, your cluster
-    name, and the service namespace. Ensure `"${WORK_DIR}"` is prefixed with `"${HOME}"`
+1. Create a VM in one of the cloud provider
```

Comment:
please remove "in one of the cloud provider". These docs should be platform agnostic.